shell.nix

```diff
@@ -0,0 +1,48 @@
+{ pkgs ? import <nixpkgs> { } }:
+let
+  inherit (pkgs) lib buildGoPackage fetchFromGitHub;
+
+  spiff = buildGoPackage rec {
+    pname = "minio-exporter";
+    version = "1.6.0-beta-1";
+    rev = "v${version}";
+
+    goPackagePath = "github.com/mandelsoft/spiff";
```

Comment:
interesting...

Reply:
/invite @mandelsoft 

as he seems to have experience with the changes in this PR.